components/molecule/selectField/README.md

```diff
@@ -0,0 +1,114 @@
+# MoleculeSelectField
+
+
+`MoleculeSelectField` is a component that wraps a composition of Label + some input (input, textarea, ...) + Validations  Messages for a [`MoleculeSelect`](https://sui-components.now.sh/workbench/molecule/select/demo) component
+
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-molecule-select-field --save
+```
+
+## Usage
+
+After importing the component `MoleculeSelectField` like this
+
+```javascript
+import MoleculeSelecyField from '@s-ui/react-molecule-select-field'
+import MoleculeSelectOption from '@s-ui/react-molecule-dropdown-option'
+
+const IconCloseTag = () => <span>x</span>  
```

Comment:
Not using this, looks like you forgot to add the required closeIcon in all examples

Reply:
There was no example with multiselection mode that is the only _mode_ that requires this icon. Added this use case!